### PR TITLE
prow: notify users if LGTM label is removed due to new pull request

### LINKED ISSUE #7159: prow should comment when removing LGTM
From an offline discussion with @krzyzacy: right now the lgtm after commit support just removes LGTM without commenting, which can confuse some users. action needs to be taken after this so we probably ought to notify users.

/area prow
/kind feature

### PR DESCRIPTION
**Pull request by qhuynh96**
* Fixes #7159

### CODE REVIEW

File: prow/plugins/lgtm/lgtm_test.go [hunk 1]
```diff
@@ -229,24 +230,165 @@ func TestLGTMComment(t *testing.T) {
 	}
 }
 
+func TestLGTMCommentWithLGTMNoti(t *testing.T) {
+	// "a" is the author, "a", "r1", and "r2" are reviewers.
```
Comment (stevekuznetsov) [inline prow/plugins/lgtm/lgtm_test.go]: go ahead and just change the string literals instead of this

Comment (BenTheElder) [reply prow/plugins/lgtm/lgtm_test.go]: yes please, or at least use longer names, I shouldn't need to reference this to read the test cases :-)

Comment (qhuynh96) [reply prow/plugins/lgtm/lgtm_test.go]: 👍 I will make changes to other tests also

**Pull request merged**